github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -3,7 +3,8 @@ presubmits:
   - name: check-up-kind-1.17-sriov
     annotations:
       k8s.v1.cni.cncf.io/networks: multus-cni-ns/sriov-passthrough-cni
-    always_run: true
+    always_run: false # only because we have run_if_changed
+    run_if_changed: "^cluster-up/cluster/kind-k8s-sriov-1.17.0/.*"
```

Comment:
We depend on the base kind too: https://github.com/kubevirt/kubevirtci/blob/master/cluster-up/cluster/kind-k8s-sriov-1.17.0/provider.sh#L45

Reply:
Right, thanks
will add it

or should we untangle it (duplicate the file to the two kind providers we have as was done for VM based providers?)